components/file_preview_modal/image_preview.jsx

```diff
@@ -25,6 +26,8 @@ export default function ImagePreview({fileInfo, canDownloadFiles}) {
         return <img src={previewUrl}/>;
     }
 
+    const svgStyleAttribute = fileInfo.extension === FileTypes.SVG && {style: {width: fileInfo.width}};
```

Comment:
This case is fine though since the `fileInfo` is required

Reply:
Fix